frontend/src/components/CostTableChecked.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div>
+    <n-button quaternary @click="handleToggleIsExpanded">
+      <div class="flex">
+        <div class="font-medium text-lg pb-1">Selected Instance</div>
+        <heroicons-outline:chevron-down class="ml-1 h-6" v-show="isExpended" />
+        <heroicons-outline:chevron-up class="ml-1 h-6" v-show="!isExpended" />
+      </div>
+    </n-button>
+
+    <cost-table
+      v-show="isExpended"
+      class="my-2"
+      virtual-scroll
+      :data-row="dataRow"
+      :isLoading="isLoading"
```

Comment:
use kebab-case when binding properties. e.g. `:is-loading`.

Reply:
This should be a bug in GitHub. I have changed it to is-loading.